### DISCUSSION TITLE
如何将一个表同步到不同的数据库

### DISCUSSION THREAD

**Discussion by sadanyoyo** [apache/flink-cdc#890] [Q&A]
主库中有一个小表，现在想将这个小表广播到其他数据库中。使用SQL的方式，写SQL语句的话，是不是会对主库订阅多次 Binlog，这样是不是会有压力？有没有一次订阅，同时同步到多个数据库的方式？

**Comment by zhongbinde** [comment]
用sql貌似不行，的确会多次多binlog，可以用api方式，分流

**Reply by gongshiyun** [reply]
请问下如何分流？有代码参考吗

**Comment by zhongbinde** [comment]
这是我测试的代码，但很多表的时候怎么让它自动化还不知道怎么写



从 Windows 版邮件发送

发件人: Shiyun Gong
发送时间: 2022年8月25日 10:52
收件人: ververica/flink-cdc-connectors
抄送: zhongbinde; Comment
主题: Re: [ververica/flink-cdc-connectors] 如何将一个表同步到不同的数据库 (Discussion #890)

请问下如何分流？有代码参考吗
